pages/_app.js

```diff
@@ -1,7 +1,14 @@
-import '../styles/globals.css'
+import GlobalStyle from '../components/GlobalStyle';
+import Layout from '../components/Layout';
+import '../styles/globals.css';
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return (
+    <Layout>
+      <GlobalStyle />
```

Comment:
```suggestion
<GlobalStyle />
    <Layout>
```
The GlobalStyled usually belong outside of the Layout :)

Reply:
Done